tests/transports-governance/test_helpers.py

```
@@ -0,0 +1,644 @@
+        return errors
+
+
+class TestScenarioBuilder:
+    """Builder for complex test scenarios"""
+
+    def __init__(self, client: GovernanceTestClient, cleanup_tracker):
+        self.client = client
+        self.cleanup_tracker = cleanup_tracker
+        self.created_entities = {"customers": [], "teams": [], "virtual_keys": []}
+
+    def create_customer(self, **kwargs) -> Dict[str, Any]:
```

Ruff (0.12.2)

Missing type annotation for `**kwargs`

(ANN003)